src/components/CreateCourse/components/AuthorItem/AuthorItem.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import Button from '../../../../common/Button/Button';
+import PropTypes from 'prop-types';
```

Comment:
to the top and add space to make a distinction between libraries and custom imports